README.md

```diff
@@ -218,7 +218,7 @@ export default NodeComponent.extend();
 
 ```hbs
 <!-- components/root.hbs -->
-<div {{did-insert (action this.didInsertNode)}} {{will-destroy (action this.willDestroyNode)}}>
```

Comment:
Hmm, why is it redundant? unless I'm misreading the example, I think these `(action` helper invocations are required in order to bind `didInsertNode` to the current component context. It would be redundant if the example code used `@action`. I'd definitely be in favor of removing these `(action` calls and adding `@action` though.

Reply:
I think I misread it I thought methodes had `@action` already in them, I updated the PR with all classes converted to es6 and using `@action`

Reply:
Perfect thank you!!